test/test_signalfd.rs

```diff
@@ -11,8 +11,7 @@ use nix::unistd;
 fn main() {
     print!("test test_signalfd ... ");
 
-    let mut mask = signal::SigSet::empty();
-    mask.add(signal::SIGUSR1).unwrap();
+    let mask: signal::SigSet = signal::SIGUSR1.into();
```

Comment:
That'd a good point. My question now is why doesn't `ppoll` take a `&SigSet` instead of by-value?